31 update and add new vignette (#34)
insightsengineering/rtables.officer: vignettes/rtables.officer.Rmd

---
title: "Introduction to {rtables.officer}"

> author: "Davide Garolini and Joe Zhu"

Author: ```suggestion
author: "Davide Garolini, Emily de la Rua and Joe Zhu"
```

## Building a Table

> The aim of this vignette is to build the following table and export as a `docx` document:

Reviewer: ```suggestion
The aim of this vignette is to build the following table and export it as a `docx` document:
```

tbl
```

We will export the `rtables` object into a `docx` file, namely `example.docx`.

```{r, echo = FALSE}
output_dir <- tempdir()
# output_dir <- "." try this line yourself
export_as_docx(tbl, file = file.path(output_dir, "example.docx"))
```

> The internal of the code convert the `rtables` object into `flextable` object, and uses `officer` package to save the `docx` output file.

Reviewer: ```suggestion
The code internally converts the `rtables` object into a flextable object and uses the `officer` package to save the output as a `docx` file.
```
